packages/vscode-extension/src/utils/markdown.ts

```diff
@@ -0,0 +1,20 @@
+import assert from 'assert'
+import { MarkdownString } from 'vscode'
+
+
+export default function (templates: TemplateStringsArray, ...args: (string | MarkdownString | any)[]) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/LinqLover/downstream-repository-mining/pullRequest?prid=8058375)
```suggestion
export default function (templates: TemplateStringsArray, ...args: (string | MarkdownString | unknown)[]) {
```